src/geop/services/src/main/scala/com/azavea/mmw/GeopServiceActor.scala

```diff
@@ -32,23 +28,57 @@ object Loader {
 
   import scala.util._
 
-  import DataFiles._
   import Histogram._
 
+  val bucket = "com.azavea.datahub"
+  val prefix = "catalog"
+  val layerName = "nlcd-tms"
+  val layerZoom = 13
```

Comment:
What's the implication of a fixed layerZoom level here and how did you come to the current value, 13?